### PR TITLE
quick consistency/cleanup fixes

### PR DESCRIPTION
**Pull request by lasers**
For `3.7.1` or `3.8` or `whatever`. I fixed column for `google_calendar` too.

### CODE REVIEW

File: py3status/modules/google_calendar.py [hunk 1]
```diff
@@ -12,12 +13,11 @@
...
         (case insensitive).
         (default [])
-    button_open: Opens the URL for the clicked on event in the default
-        web browser.
+    button_open: Opens the event URL in the default web browser.
         (default 3)
     button_refresh: Refreshes the module and updates the list of events.
         (default 2)
-    button_toggle: The button used to toggle what data is displayed for each event.
+    button_toggle: Toggles a boolean to hide/show the data for each events.
```
Comment (tobes) [inline py3status/modules/google_calendar.py]: __event__ or else different wording

File: py3status/modules/google_calendar.py [hunk 2]
```diff
@@ -30,24 +30,25 @@
...
+        with 'button_toggle' based on the value of 'is_toggled'.
+        (default '[\?color=event {summary}][\?if=is_toggled  ({start_time}
+         - {end_time}, {start_date})|[\?if=location  ({location})]
+        [{format_timer}]]'
     format_notification: The format for event warning notifications.
         (default '{summary} {start_time} - {end_time}')
     format_separator: The string used to separate individual events.
         (default ' \| ')
-    format_time: The format for time related format placeholders (except {format_timer}).
+    format_time: The format for time-related placeholders except {format_timer}.
```
Comment (tobes) [inline py3status/modules/google_calendar.py]: except \`{format_timer}\`.  

will look nicer in the docs

File: py3status/modules/google_calendar.py (hunk 1 above)
Comment (lasers) [reply py3status/modules/google_calendar.py]: It's finished. That's what I was changing just now... Ha.

File: py3status/modules/google_calendar.py (hunk 2 above)
Comment (lasers) [reply py3status/modules/google_calendar.py]: Okay. Coming up soon.

**Comment by tobes** [comment]
do you want to keep the documentation fixes separate from the code changes as then I can merge them much quicker

**Comment by lasers** [comment]
Okay. I'll undo the code changes.

**Comment by lasers** [comment]
Ping.

**Comment by lasers** [comment]
Do you want 2nd commit for code? Or PR?

**Comment by ultrabug** [comment]
Thanks @lasers I'll go ahead on this 👍

**Pull request merged**